config/initializers/sentry.rb

```diff
@@ -7,6 +7,6 @@
     config.release = ENV["COMMIT_SHA"]
     config.traces_sample_rate = 0.1
     config.profiles_sample_rate = 0.1
-    config.active_job_report_on_retry_error = true
+    config.rails.active_job_report_on_retry_error = true
```

Comment:
Looks good to me [based on what's in the gem](https://github.com/getsentry/sentry-ruby/blob/818b6c0f6c2c717a697ee67ecaf0ac78feea842f/sentry-rails/lib/sentry/rails/active_job.rb#L92).